readme.md

```diff
@@ -109,6 +109,7 @@ The minimum requirements are for the device platform to support the following:
 - **Support SHA-256** (optional): necessary to generate the secure token for authenticating the device with the service. Different authentication methods are available and not all require SHA-256.
 - **Have a Real Time Clock or implement code to connect to an NTP server**: necessary for both establishing the TLS connection and generating the secure token for authentication.
 - **Having at least 64KB of RAM**: the memory footprint of the SDK depends on the SDK and protocol used as well as the platform targeted. The smallest footprint is achieved targeting microcontrollers.
+- **Having at least 4KB set for incoming SSL max content length buffer**: For some TLS libraries, this may be a configurable option and default may have been set as **4KB** for low memory footprint devices. During TLS handshake, IoT Hub service will send Server Hello which includes IoT Hub server side certificates. During **renewal** of IoT Hub server side certificates, check will be made to prevent this exceeding 4KB limit so that existing devices which are set for 4KB limit continue to work as before after renewal.
```

Comment:
@anhashia  - I assume you meant 4KB of RAM. Please explicitly add that.
@jebrando - please take a look at the readme contents here in general for anything else we need to add in this iteration.

Reply:
Added RAM in the text